dokka-subprojects/core/src/main/kotlin/org/jetbrains/dokka/transformers/documentation/DefaultDocumentableMerger.kt

```diff
@@ -116,9 +116,11 @@ public class DefaultDocumentableMerger(context: DokkaContext) : DocumentableMerg
                 listOf(actuals to actuals.flatMap { it.sourceSets }.toSet())
             } else expects.map { expect ->
                 val actualsForGivenExpect = actuals.filter { actual ->
-                    dependencyInfo[actual.sourceSets.single()]
+                    actual.sourceSets.all {
```

Comment:
could you please add some info in PR description / comment in code, on why this should be like this?